backend/src/emissions/schema.py

```diff
@@ -278,7 +278,14 @@ def resolve_workinggroups(self, info, **kwargs):
     def resolve_workinggroup_users(self, info, **kawrgs):
         """Returns the users of a certain working group."""
         id = info.context.user.working_group.id
-        return CustomUser.objects.filter(working_group__id = id)
+        return CustomUser.objects.filter(working_group__id=id)
+
+    @login_required
+    @representative_required
+    def resolve_join_requests(self, info, **kwargs):
```

Comment:
I think it would be helpfull to just show pending requests. But we can solve this in the frontend aswell so I think we are fine :)

Reply:
ok, then let's leave it like this for now. If it's necessary to filter by status we can fix it later.